contribution-and-developer-guidelines.md

```diff
@@ -0,0 +1,102 @@
+# CoatRack Contribution and Developer Guidelines
+
+
+
+## Introduction
+
+#### Possibilities to Contribute
+
+Hello, dear prospective contributor. You can support CoatRack in many ways:
+
+- Report bugs using GitHub's issue tracker.
+- Discuss the current state of the code in open Pull Requests.
+- Submit a bugfix.
+- Propose new features and improvement ideas.
+- Improve the documentation.
+- Clean up code.
+- Add tests.
+- Help us to fulfill the [FIWARE requirements](https://fiware-requirements.readthedocs.io/en/latest/).
```

Comment:
I think this wording here could be misleading.
The contributor will fill up also the FIWARE requirements directly?

Reply:
As we have already fullfilled many, I would say: 
```suggestion
- Help us to fulfill all [FIWARE requirements](https://fiware-requirements.readthedocs.io/en/latest/).
```